ingest_revisions_dataflow/ingest_utils/wikipedia_revisions_ingester.py

```diff
@@ -33,101 +33,76 @@ def __init__(self, data_reset_path, data_paths, revision_reset_path, out_file):
     self.data_paths = data_paths
     self.revision_reset_path = revision_reset_path
     self.page_data = {}
-    self.out_file = out_file
 
   def startElement(self, name, attrs):
     self.xml_path.enter(name)
-    # print('# START: ' + str(self.xml_path))
 
   def endElement(self, name):
-    # print('# END: ' + str(self.xml_path))
+    # TODO(nthain): Wrap the inside of the top ifs in a try-catch
+    # to determine when things go wrong.
     if self.xml_path.element_path_eq(self.revision_reset_path):
-       if int(self.page_data['page_namespace']) in TALK_PAGE_NAMESPACE:
-          if 'revisions' not in self.page_data: self.page_data['revisions'] = [] 
-          if 'user_ip' in self.data:
-             self.data['user_id'] = -1
-             self.data['user_text'] = self.data['user_ip'] 
-          if not('text' in self.data):
-             self.data['text'] = ""
-          if not('user_id' in self.data):
-             self.data['user_id'] = None
-             self.data['user_text'] = None
-          self.page_data['revisions'].append(self.data)
-          self.data = {} 
+      if 'user_ip' in self.data:
+         self.data['user_id'] = -1
+         self.data['user_text'] = self.data['user_ip']
+      self.data['page_id'] = self.page_data['page_id']
+      self.data['page_namespace'] = self.page_data['page_namespace']
+      self.data['page_title'] = self.page_data['page_title']
+      if not('text' in self.data):
+         self.data['text'] = ""
+      if not('user_id' in self.data):
+         self.data['user_id'] = None
+         self.data['user_text'] = None
+      #if self.data['page_namespace'] in TALK_PAGE_NAMESPACE:
```

Comment:
Remove comment?

Reply:
Done. Now it only writes talk pages which should save file size enormously.